core/spec/helpers/image_helper_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+module MnoEnterprise
+  describe ImageHelper do
+    
+    IMAGES_LOCATION = "/app/assets/images/mno_enterprise"
+
+    describe "#main_logo_white_bg_path" do
+      it "should return the engine's main-logo filename, when there are no logos"  do
+        expect(helper.main_logo_white_bg_path).to eq("mno_enterprise/main-logo.png")
+      end
+      
+      it "should return the engine's main-logo full path when there are no logos"  do
+        path = "#{MnoEnterprise::Engine.root}#{IMAGES_LOCATION}/main-logo.png"
+        expect(helper.main_logo_white_bg_path(true)).to eq(path)
+      end
```

Comment:
Some tips from http://www.betterspecs.org
* use contexts to reduce duplication and make it easier to read
* Don't use should
* you're missing a test case

Something along the lines of:
```ruby
context 'with an app main-logo-whitebg' do
   before { #stub file exist for main-logo-whitebg } 
   it "returns the app's main-logo-whitebg"
end
context 'with an app main-logo' do
   before { #stub file exist for main-logo } 
   it "returns the app's main-logo"
end
context 'when there are no logos' do
  it "returns the engine's main-logo filename"
  it "returns the engine's main-logo full path"
end
```
(wording to be improved)